### PR TITLE
chapter9 - 3d

### PR DESCRIPTION
**Pull request by drelaptop**

### CODE REVIEW

File: zh/3d/terrain.md [hunk 1]
```diff
@@ -65,31 +51,26 @@ Terrain::DetailMap b("road.dds");
...
+
+### 获取地形高度
 
-### Get Terrain Height
-Use the method __Terrain::getHeight(float x, float z, Vec3 * normal= nullptr)__ to
-get the specified position's height. This method is very useful when you want to
-put a `Sprite3D` object or any `Node` on the terrain's surface.
+使用方法 `Terrain::getHeight(float x, float z, Vec3 * normal= nullptr)` 可以获取到地形的高度, 当你想把一个节点对象放到地形表面的时候, 这个方法会非常有用.
 
 ### Ray-Terrain intersection test
```
Comment (drelaptop) [inline zh/3d/terrain.md]: I can't understand this paragraph means, title is `Ray-Terrain intersection test`, but most of contents talk about  LOD policy
```
A __Ray-Terrain__ intersection test will calculate the intersection point by giving
  a specified position.

__Terrain::CrackFixedType::SKIRT__ will generate four skirt-like meshes at each
		 
__Terrain::CrackFixedType::SKIRT__ will generate four skirt-like meshes at each
```
is it unfinished? @slackmoehrle

Comment (slackmoehrle) [reply zh/3d/terrain.md]: This text came from an employee that no longer works here. We may need to ask the current 3D developer to take a look at this. I am not sure who is the current 3D dev, do you know?

Comment (drelaptop) [reply zh/3d/terrain.md]: I don't know either. I will beg @minggo review, after this chapter finished

Comment (slackmoehrle) [reply zh/3d/terrain.md]: I think I recall the previous developer being named Tony

**Pull request merged**